.travis.yml

```diff
@@ -7,10 +9,27 @@ php:
   - 7.0
   - 7.1
   - nightly
-  - hhvm
 
 matrix:
+  fast_finish: true
+  include:
+    - php: hhvm
+      sudo: true
+      dist: trusty
+      group: edge # Use edge image until the next travis CI image update
+      addons:
+        apt:
+          packages:
+            - mysql-server-5.6
+            - mysql-client-core-5.6
+            - mysql-client-5.6
+      services:
+        - mysql
+        - postgresql
+        - redis-server
+      env: CUBRID_VERSION=9.3.0/CUBRID-9.3.0.0206 CUBRID_PDO_VERSION=9.3.0.0001
   allow_failures:
+    - php: hhvm
```

Comment:
There is a testing setup issue which is why I've set hhvm to  allowed failure.

Testing on Trusty does not have a mysql `travis` user
You can read the reason for root in the travis docs
https://docs.travis-ci.com/user/database-setup/#MySQL-5.6

I'm not sure if there is a way to create a valid mysql `travis` user on Trusty that would have the same permissions and equivalance as the mysql `travis` user on the Precise containers. On Precise, it's part of the build no user creation needed. Unfortunately, yii2 has hard coded the travis user in the test setup. If you have suggestions on working around that detail I'll make the necessary changes